web/html/src/manager/image-build.js

```diff
@@ -20,7 +20,7 @@ const typeMap = {
 };
 
 const msgMap = {
-  "unknown_error": t("Some unknown error has been occured."),
+  "unknown_error": t("Some unknown error has been occurred."),
```

Comment:
I think the correct phrasing would be: `Some unknown error has occurred.` or better `An unknown error has occurred.`

Reply:
fixed